schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow",
+      "name": "failure",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "source": {
+            "type": ["string", "null"],
+            "description": "Source of the error. If the failure happened during enrichment, it'll be the enriched field - for example `user_ipaddress`, `unstruct` or `contexts`"
+        },
+        "failureType": {
+            "type": "string"
+        },
+        "errors": {
+            "type": "array",
+            "items": {
+                "type": "object"
+            },
+            "description": "A list of errors encountered, and supporting information about the error."
+        },
+        "schema": {
+					"type": ["string", "null"],
+					"description": "The schema for the object which caused the failure, if it was self-descrbing."
+				},
+        "data": {
+            "type": ["object", "null"],
+            "additionalProperties": true,
+            "description": "The original data object which caused the failure."
+        },
+        "timestamp": {
+            "type": "string",
+            "description": "Timestamp at which the failure occurred",
+            "format": "date-time"
+        },
+        "producingAppName": {
+            "type": "string",
+            "description": "Name of the app which produced the failure"
+        },
+        "producingAppVersion": {
+            "type": "string",
+            "description": "Version of the app which produced the failure"
```

Comment:
I'm confused, are these trackers? App_ids? Or the part of the pipeline?

Reply:
Component of the pipeline

Reply:
I see the confusion though. Worth considering alternative names I guess. Any suggestions?

Reply:
`component` / `component_version`? I think we call this artifact at the moment but component is likely more clear.